src/lib.rs

```diff
@@ -12,7 +12,7 @@
 //! [`kurbo`]: https://crates.io/crates/kurbo
 
 #![cfg_attr(all(not(feature = "std"), not(test)), no_std)]
-#![warn(clippy::doc_markdown)]
+#![warn(clippy::doc_markdown, unused_qualifications)]
```

Comment:
We should have a `TODO` (probably in `Cargo.toml`?) to move to the `lints` table once our MSRV allows it.

Reply:
Added and re-pushed.